CruCentralCoast/Community/Missions/MissionDetailsVC.swift

```diff
@@ -30,7 +33,17 @@ class MissionDetailsVC: UIViewController {
     
     @IBAction func learnMore() {
         // TODO
-        self.presentAlert(title: "Learn More", message: "Coming Soon...")
+        
+        if let url = URL(string: missionURLString!) {
+            UIApplication.shared.open(url, options: [:], completionHandler: nil)
+//            let config = SFSafariViewController.Configuration()
```

Comment:
Remove these commented lines of code and the todo comment above

Reply:
thanks!